app-misc/piper/piper-0.6.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit meson python-single-r1 xdg
+
+DESCRIPTION="GTK configuration application for libratbag"
+HOMEPAGE="https://github.com/libratbag/piper"
+SRC_URI="https://github.com/libratbag/piper/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	${PYTHON_DEPS}
+	virtual/pkgconfig
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/flake8[${PYTHON_USEDEP}]
+		')
```

Comment:
We try not to, 
https://projects.gentoo.org/python/guide/distutils.html?highlight=flake#undesirable-test-dependencies

and the test phase fails anyway.